plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -495,6 +501,14 @@ class MarathonStore extends GetSetBaseStore {
         };
       }
 
+      if (item instanceof Application) {
+        map[item.getName().toLowerCase()] = {
+          health: item.getHealth(),
```

Comment:
do we really need adding `health` and `snapshot` along with images?

Reply:
The alternative is changing the marathon store test.

Reply:
That was suggested by @juliangieseke , and my assumption is that if frameworks can have health and snapshot, then pseudo-frameworks can have them as well.

Reply:
Those are catalog packages, not frameworks

Reply:
Aren't catalog packages usually frameworks?

Reply:
nope, a catalog package it's just a configurable Marathon application definition with some artefacts like tar files and docker images. It may or may not be a framework. 

Framework in DCOS and Mesos is an application that schedules mesos tasks when needed. In this regard Marathon is a Framework, Spark and Kafka are as well, but `ngnix` isn't for example.

http://mesos.apache.org/documentation/latest/frameworks/

Reply:
In theory, yes, but in practice, aren't the majority of catalog packages frameworks?

Reply:
I would argue that in practice the majority of packages are _not_ frameworks.

Reply:
Then we have one more reason to allow non-frameworks to have images.

Reply:
images yes, my question was regarding `health` and `snapshot` fields

Reply:
Adding health and snapshot was @juliangieseke 's suggestion and I added them with the assumption that Applications also have them.